drivers/test-fw/Cargo.toml

```diff
@@ -10,17 +10,21 @@ edition = "2021"
 caliptra-drivers = { path = "..", features=["emu"] }
 caliptra-registers = { path = "../../registers" }
 caliptra-kat = { path = "../../kat" }
+caliptra-test-harness = { path = "../../test-harness" }
 cfg-if = "1.0.0"
 
 [features]
 emu = []
 
+# Tells the test_suite macro that this is a ROM image, not a runtime image
+rom = []
```

Comment:
Can you explain why this is necessary?

Reply:
It's not, I accidentally removed it in the wrong commit. Fixed.